src/metrics/cyclomatic.rs

```diff
@@ -1,22 +1,27 @@
+use crate::checker::Checker;
+use crate::*;
 use serde::ser::{SerializeStruct, Serializer};
 use serde::Serialize;
 use std::fmt;
 
-use crate::checker::Checker;
-use crate::*;
-
 /// The `Cyclomatic` metric.
 #[derive(Debug, Clone)]
 pub struct Stats {
+    self_cyclomatic: f64,
```

Comment:
The role of this new field is unclear for me and its name isn't really self-explanatory.
So it'd be nice to add a comment to explain why we need it and why we can't simply use `cyclomatic`.

Reply:
Sorry but the comment is still unclear for me:
 - the concept of `line 374` is a bit fuzzy in the time line, don't you think ?
 - my impress is that this field is useless (this is reinforced by the fact you're almost duplicating code for the both fields), and your comment doesn't explain anything I don't know, here you're telling me to read the c_unit_after test and try to figure out myself what's wrong when we don't save this self_cyclomatic... And if someone read your comment in 10 months he will have to spend some time to figure why this value is useful.
So here, you must help me to understand very quickly why you MUST save this value, with a small example or a small explanation.